project.php

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Project Manager - Project Name</title>
+    <script src="https://cdn.tailwindcss.com"></script>
+</head>
+<body class="h-full">
+<header class="p-3 bg-teal-50 flex justify-between">
+    <h1 class="sm:text-5xl text-4xl"><a href="index.php">Project Manager</a></h1>
+    <div class="pr-3 flex">
+        <a href="project.php" class="p-3 bg-slate-300 rounded-l-lg border-y border-l">🇬🇧</a>
+    </div>
+</header>
+<main class="p-3">
+
+    <?php
+        require_once 'src/Services/DatabaseService.php';
+        require_once 'src/Entities/ProjectEntity.php';
+        require_once 'src/Services/ProjectDisplayServices.php';
+        require_once 'src/Models/ProjectsModel.php';
+        require_once 'src/Services/ClientDisplayService.php';
+        require_once 'src/Models/ClientsModel.php';
+        require_once 'src/Services/ProjectLinkService.php';
+        require_once 'src/Services/DeadlineDateService.php';
+        require_once 'src/Models/TasksModel.php';
+        require_once 'src/Services/TaskLinkService.php';
+
+        $db = DatabaseService::connect();
+
+        $projectsModel = new ProjectsModel($db);
+        $tasksModel = new TasksModel($db);
+        $ClientsModel = new ClientsModel($db);
+
+        $project = $projectsModel->getProjectById($idLink);
+        $projectTitle = ProjectDisplayService::displayProject($project);
+        $clientID = $project->client_id;
+        $projectDeadline = DeadlineDateService::reformatDateUK($project->deadline);
+
+        $client = $ClientsModel->getClientById($clientID);
+        $clientLogo = $client->logo;
+        $clientTitle = ClientDisplayService::displayClient($client);
+
+        $displayUserNameByProjectId = $tasksModel->getUsersByProjectId($idLink);
+
+        echo "
+            <div class=\"flex justify-between mb-3\">
+                <h2 class=\"text-4xl font-bold mb-2\">$projectTitle$projectDeadline
+                    <a href=\"index.php\" class=\"text-base text-blue-600 hover:underline ms-3\">Return to all projects</a>
+                </h2>
+                <div class=\"flex items-center gap-3\">
+                    <h3 class=\"text-3xl font-bold\">$clientTitle</h3>
+                    <img class=\"w-[50px]\" src=$clientLogo alt=\"client logo\" />
+                </div>
+            </div>
+            ";
```

Comment:
As much as possible you should try to avoid echoing big chunks of html like this. It makes it much harder to read, and you end up having to put loads of \ in the string to make it happy.

Instead this code should just be in your HTML with php tags added in to display the data you need.

The only time I would echo big chunks like this is when they are part of a foreach loop or an if statement